app/Providers/SettingsServiceProvider.php

```diff
@@ -87,6 +87,24 @@ public function boot(ConfigRepository $config, Encrypter $encrypter, SettingsRep
                 }
             }
 
+            switch (strtolower($value)) {
+                case 'true':
+                case '(true)':
+                    $value = true;
+                    break;
+                case 'false':
+                case '(false)':
+                    $value = false;
+                    break;
+                case 'empty':
+                case '(empty)':
+                    $value = '';
+                    break;
+                case 'null':
+                case '(null)':
+                    $value = null;
+            }
+
```

Comment:
This is fantastic. Maybe this should be put in a separate (global?) function.

Reply:
Its actually copy/pasted from Laravel's `env()` function, I put it here because it was causing issues with the true/false stuff.

Shouldn't be needed anywhere else to my knowledge.